bin.src/ci_imsim_run.py

```diff
@@ -92,6 +92,7 @@ def run(self, currentState: BuildState):
             "--skip-existing",
             "--save-qgraph", os.path.join(self.runner.RunDir, QGRAPH_FILE),
             "--config", f"calibrate:deblend.useCiLimits={not self.arguments.no_limit_deblend}",
+            "--config", f"reprocessVisitImage:deblend.useCiLimits={not self.arguments.no_limit_deblend}",
```

Comment:
Do we really need this? I removed the `calibrate` line above, and I'm not sure it's really necessary here. Maybe it is necessary for `reprocessVisitImage`?

Reply:
Yeah, I'm not sure how much faster it makes ci_imsim, but I assume it was added for a good reason initially and I imagine that reasoning would apply to `reprocessVisitImage` as well.